src/main/java/com/google/edith/servlets/Receipt.java

```diff
@@ -42,10 +42,44 @@ public Receipt(
     this.items = items;
   }
 
+  public String toString() {
+    return (this.userId
+        + " "
+        + this.storeName
+        + " "
+        + this.date
+        + " "
+        + this.name
+        + " "
+        + this.fileUrl
+        + " "
+        + this.totalPrice);
```

Comment:
Could we use a String template that gets formatted instead of appending spaces?

```
private static final String RECEIPT_STRING_TEMPLATE = "%s %s %s %s %s";
return String.format(RECEIPT_STRING_TEMPLATE, storeName, date, ...);